react/RuiBreadcrumbsReact/src/RuiBreadcrumbs.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+* Copyright Deloitte Digital 2019
+*
+* This source code is licensed under the BSD-3-Clause license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+import React from 'react';
+import '@rhythm-ui/rui-breadcrumbs';
+
+export class RuiBreadcrumbs extends React.Component {
```

Comment:
@grmillsy1 Following our discussion on the web component abstraction, can you please add a <RuiBreadCrumbs.Crumb> sub component here to abstract the `slot='crumb'` from children

Reply:
@jnelssonsmith Yeah I really liked this approach. I have updated this in my latest version just haven't pushed it yet..will update you when I have!